### PR TITLE
Config table

### PR DESCRIPTION
**Pull request by jayajoseph1**
Exception Added

### CODE REVIEW

File: src/main/java/com/ust/sourcecourse/configuration/excpection/ResourceNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.ust.sourcecourse.configuration.excpection;
```
Comment (shijukbabu-ust) [inline src/main/java/com/ust/sourcecourse/configuration/excpection/ResourceNotFoundException.java:1]: Correct the package name. Take latest code from dev. There is already ResourceNotFoundException class added, but in exceptions package. You can use the same and delete this package.

**Review by shijukbabu-ust — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties
Comment (shijukbabu-ust) [inline src/main/resources/application.properties:1]: Don't change this file yet.

File: src/main/java/com/ust/sourcecourse/configuration/controller/SourcePocController.java
Comment (shijukbabu-ust) [inline src/main/java/com/ust/sourcecourse/configuration/controller/SourcePocController.java:1]: Delete this class

**Review by shijukbabu-ust — CHANGES_REQUESTED** [review]

File: src/main/java/com/ust/sourcecourse/configuration/controller/SourcePocController.java
Comment (jayajoseph1) [reply src/main/java/com/ust/sourcecourse/configuration/controller/SourcePocController.java:1]: resolved

File: src/main/resources/application.properties
Comment (jayajoseph1) [reply src/main/resources/application.properties:1]: resolved

**Review by shijukbabu-ust — APPROVED** [review]

**Pull request merged**